src/Database/Esqueleto/Internal/Internal.hs

```diff
@@ -137,12 +139,6 @@ where_ expr = Q $ W.tell mempty { sdWhereClause = Where expr }
 -- and tuple-joins do not need an 'on' clause, but 'InnerJoin' and the various
 -- outer joins do.
 --
--- Note that this function will be replaced by the one in
--- "Database.Esqueleto.Experimental" in version 4.0.0.0 of the library. The
--- @Experimental@ module has a dramatically improved means for introducing
--- tables and entities that provides more power and less potential for runtime
--- errors.
```

Comment:
Why is this removed?

Reply:
@parsonsmatt This was removed when we were trying to figure out why docs weren't building in 3.3.4.0. My bad, should be re-added.